flag_exposure_outliers.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import print_function, division
+
+# fix for DISPLAY variable issue
+import matplotlib
+matplotlib.use('Agg')
+
+import numpy as np
+from scipy import stats
+from astropy.io import fits
+import fitsio
+import matplotlib.pyplot as plt
+import os
+import glob
+
+import lmfit
+import galsim
+import piff
+import copy
+
+from matplotlib.backends.backend_agg import FigureCanvasAgg
+from matplotlib.figure import Figure
+
+
+
+def flag_exposure_outliers():
+    core_directory = os.path.realpath(__file__)
+    program_name = core_directory.split("/")[-1]
+    core_directory = core_directory.split("/{0}".format(program_name))[0]
+    source_directory = np.load("{0}/source_directory_name.npy".format(core_directory))[0]
+    original_exposures = glob.glob("{0}/*".format(source_directory))
+    original_exposures = [original_exposure.split("/")[-1] for original_exposure in original_exposures]
```

Comment:
since I see these lines everywhere, I wonder if you couldn't make these come from a common function call, to clean up the code a bit?